smartclide-monitoring/pom.xml

```diff
@@ -38,7 +38,7 @@
                         <image>openjdk:11.0.10-jre-slim-buster</image>
                     </from>
                     <to>
-                        <image>ghcr.io/eclipse-researchlabs/smartclide/${project.artifactId}:latest</image>
+                        <image>ghcr.io/eclipse-researchlabs/smartclide/${project.artifactId}</image>
```

Comment:
Why do we have to remove this?

I would like to keep the `latest` tag.

Reply:
During my tests, I have realized that if we keep the tag `latest` in pom file, then by default it takes `latest` for final docker image url. We wanted to have release name so I had to remove it.

Reply:
But isn't that what `-Djib.to.tags="${IMAGE_TAG}"` is for?

So it creates two image tags, `latest` and `commit-sha` (**or** `tag_name`)

See [example in the documentation](https://github.com/GoogleContainerTools/jib/tree/v2.8.0-gradle/jib-maven-plugin#example)

> Is pushed to localhost:5000/my-image:built-with-jib, localhost:5000/my-image:tag2, and localhost:5000/my-image:latest

If we keep `<image>ghcr.io/eclipse-researchlabs/smartclide/${project.artifactId}:latest</image>` and also specify `-Djib.to.tags="${IMAGE_TAG}"` it should result in those two image tags.

Or am I missing something?